fedora/Dockerfile.armv7hf.tpl

```diff
@@ -33,26 +33,24 @@ RUN curl -SLO "http://resin-packages.s3.amazonaws.com/tini/v$TINI_VERSION/#{TINI
     && chmod +x tini \
     && mv tini /sbin/tini
 
+# Install Systemd
 ENV container docker
 
-RUN systemctl mask \
-        dev-hugepages.mount \
-        sys-fs-fuse-connections.mount \
-        sys-kernel-config.mount \
-        display-manager.service \
-        getty@.service \
-        systemd-logind.service \
-        systemd-remount-fs.service \
-        getty.target \
-        graphical.target \
-        console-getty.service \
-        systemd-vconsole-setup.service
-
-COPY entry.sh /usr/bin/
+# We only want few core services run in the container.
+RUN find /etc/systemd/system \
+        /lib/systemd/system \
+        -path '*.wants/*' \
+        -not -name '*journald*' \
+        -not -name '*udevd*' \
+        -not -name '*systemd-tmpfiles*' \
+        -not -name '*systemd-user-sessions*' \
+        -exec rm \{} \;
+
+COPY entry.sh /usr/bin/entry.sh
 COPY launch.service /etc/systemd/system/launch.service
 
-RUN systemctl enable launch.service systemd-udevd
+RUN systemctl enable /etc/systemd/system/launch.service
```

Comment:
You don't need the full path of the service here

Reply:
updated